src/applications/personalization/dashboard/tests/utils/appeals-helpers.unit.spec.jsx

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
+
+import {
+  formatDate,
+  getAojDescription,
+  getDocketName,
+  getHearingType,
+} from '../../utils/appeals-helpers';
+
+describe('formatDate', () => {
+  it('should format date', () => {
+    expect(formatDate('2/6/2024')).to.equal('February 06, 2024');
+  });
+});
+
```

Comment:
another test to verify behavior when a bad string is entered could be helpful here